src/commcare_cloud/commands/deploy.py

```diff
@@ -38,21 +41,33 @@ class Deploy(CommandBase):
         shared_args.BRANCH_ARG,
     )
 
+    def modify_parser(self):
+        environment = get_environment(sys.argv[1])
```

Comment:
I don't really like how this works - needing to know the env while still creating the arg parser